docs/Cliente REST/2-Axios.md

```diff
@@ -10,27 +12,41 @@ Para instalar digite os comandos abaixo:
 
 > yarn add axios
 
+## Entendo como funciona uma api 
 
-## Entendo como funciona uma Api Rest
+Antes de colocamos a mão no codigo devemos saber que uma api rest tem regras, ou seja dependendo da situação sera necessario usar um metodo **X** e em outro momento **Y**. Então que tal conhecer esse metodos.
```

Comment:
Na verdade, essas "regras" (Que na verdade são mais para o lado de opções) são definições do HTTP, REST é uma "padronização" para usar essas coisas do HTTP.

APIs, por exemplo, não necessariamente precisam ser uma interface HTTP, apenas precisam expor certas funcionalidades/dados. Um exemplo, é a API do VSCode para usos em extensões, você consegue usar localmente sem nem usar HTTP.

Em cima disso, acho que seria interessante que fosse separado esse entendimento. Se você pegar a descrição do axios, por exemplo, você vê que não se cita API, mas sim algo próximo como cliente HTTP (O que realmente soa mais correto):

`Promise based HTTP client for the browser and node.js`

Reply:
Acredito que agora fique mais claro fiz uma alteração e uma nova sessão pra separar esses conceitos